src/latexgen.cpp

```diff
@@ -346,11 +349,12 @@ static void writeLatexMakefile()
   }
   else // use pdflatex for higher quality output
   {
-    t << "LATEX_CMD=" << latex_command << endl
+    t << "LATEX_CMD:=" << latex_command << endl
       << endl;
     t << "all: refman.pdf" << endl << endl
       << "pdf: refman.pdf" << endl << endl;
-    t << "refman.pdf: clean refman.tex" << endl;
+    t << "refman.pdf: refman.tex" << endl;
+    t << "\trm -f $(CLEAN_FILES)" << endl;
```

Comment:
Why removing here the "clean" dependency and thus duplicating the code?

Reply:
In general, there shouldn't be a prerequisite `clean` - since if executing make by e.g. `make -j 4` for parallel jobs, then target `clean` may be executed at last, and hence important files could be removed as well.

Here, it does not matter... but I think it is a better style. But of course, it is only my opinion.